pkg/sloop/ingress/kubewatcher.go

```diff
@@ -270,6 +270,11 @@ func (i *kubeWatcherImpl) reportAdd(kind string) func(interface{}) {
 
 func (i *kubeWatcherImpl) reportDelete(kind string) func(interface{}) {
 	return func(obj interface{}) {
+		delObj, ok := obj.(cache.DeletedFinalStateUnknown)
```

Comment:
Is there a chance that delete object is in the format that other Add or Update objects are @duke-harlan ?

Reply:
@sana-jawad the object passed in could me any k8s object. This cast returns ok = true if the object is of that type, so the only time this has any affect is when it is a DeletedFinalStateUnknown. Otherwise it passes through as before.